test/test.js

```diff
@@ -449,4 +449,20 @@ test('should handle max safe integer', t => {
 test('should throw exception with invalid input', t => {
   // eslint-disable-next-line no-undefined
   t.throws(function() { currency(undefined, { errorOnInvalid: true }); }, Error, 'Threw exception');
-});
\ No newline at end of file
+});
+
+test('should allow creation from cents', t => {
```

Comment:
Some of the test assertions can be simplified to use `intValue` and `value` properties instead of strings. Those values should be the same for `fromCent` values and regular values of the same precision:

```js
let decimals = currency(1.23);
let fromCents = currency(123, { fromCents });
decimals.value === fromCents.value; // true
decimals.intValue === fromCents.intValue; // true
```

I would also like to see these tests broken out so it's easier to tell when a specific assertion has broken:

```js
it('should parse cents from a string when using fromCents option', t => {
  let c1 = currency('123', { fromCents })
  t.is(c1.value, 1.23)
  t.is(c1.intValue, 123)
})

it('should parse cents from a number when using fromCents option', t => { ... })
it('should truncated decimals from a string when using fromCents option', t => { ... })
it('should truncated decimals from a number when using fromCents option', t => { ... })
```

Other tests should cover the default value.